uk_bin_collection/uk_bin_collection/councils/NorthumberlandCouncil.py

```
@@ -1,7 +1,7 @@
-import time
 import datetime
-
+import time
 from datetime import datetime
```

Ruff (0.14.0)

Redefinition of unused `datetime` from line 1

(F811)